### ISSUE TITLE
When using the lib with WebPack 2 (Angular 4) getting Uncaught ReferenceError: require is not defined

### ISSUE THREAD

**Issue by born2net** [Leonidas-from-XIV/node-xml2js#383]
When using the lib with WebPack 2 (Angular 4) getting Uncaught ReferenceError: require is not defined

```
Uncaught ReferenceError: require is not defined
  (anonymous function)	
  (anonymous function)	
  webpackJsonp.14.module.exports	
  212	
  __webpack_require__	
  520	
  __webpack_require__	
  webpackJsonpCallback	
  (anonymous function)	
```

I created a small project with Angular and WebPack (uses Angular cli) and I would be happy to pay someone to help me change this lib to be flat file, no require() lib,
https://github.com/born2net/cli-filipesilva

anyway to bundle this lib so its a flat file (like umd) targeting the browser with no require statements?

regards

Sean.

**Comment by sam-s4s** [comment]
Yeah, I'm using Rollup (and sometimes Webpack), and that also doesn't have require...

I get the feeling this is an older project, and probably won't be updated for modern development?

ie. it's obsolete / deprecated ?

(for it to be useful now, we'd need it to work with ES6 import)

**Comment by penrique** [comment]
Same thing here

**Comment by Leonidas-from-XIV** [comment]
> I get the feeling this is an older project, and probably won't be updated for modern development?

This is a node project, and node does have `require`. If you try to use it on non-node platforms then I cannot really help you, there might be dragons.
